packages/sdks/web-component/src/lib/descope-wc/DescopeWc.ts

```diff
@@ -1648,13 +1649,20 @@ class DescopeWc extends BaseDescopeWc {
       ),
     ).filter((ele) => ele !== submitter);
 
-    const restoreComponentsState = () => {
+    const restoreComponentsState = async () => {
```

Comment:
Do we need to use awaut now when calling it?

Reply:
I don't think so
It's just for getting the config and it's already loaded